### ISSUE TITLE
4x slower read_csv than Pandas

### ISSUE THREAD

**Issue by zakneutron** [modin-project/modin#867]
### System information
- **OS Platform and Distribution Windows 10
- **Modin installed from pip install via anaconda
- **Modin version**: 0.6.2
- **Python version**: 3.7.4
- **Exact command to reproduce**:

CSV file size = 497 MB (521,449,891 bytes)
Shape of Dataframe = (543427, 70)
i5 laptop -8GB RAM

### Describe the problem

###Pandas ONLY read_csv import###

import pandas as pd
import time as time
s = time.time()
df = pd.read_csv("testresults.csv")
e = time.time()
print("Pandas Loading Time = {}".format(e-s))

Pandas Loading Time = 8.879479885101318

import modin.pandas as pd
import time as time
s = time.time()
df = pd.read_csv("testresults.csv")
e = time.time()
print("Modin Pandas Loading Time = {}".format(e-s))

Modin Pandas Loading Time = 26.586891174316406

**Comment by devin-petersohn** [comment]
Hi @zakneutron, thanks for posting!

Since you are using Windows, the last release for Dask did not have `read_csv` implemented in parallel. There should be a warning that says something like `UserWarning: `read_csv` is not yet implemented.` and `UserWarning: Distributing pandas.DataFrame, this may take some time.`. 

I have implemented `read_csv` for Dask within the last couple of weeks, so if you install from the GitHub master branch, it will be there. I will be releasing in the next few days.

Here is how to install from the GitHub master branch to use it now:
```
pip install git+https://github.com/modin-project/modin
```

**Comment by devin-petersohn** [comment]
Closing as we have released and this was already resolved.

**Comment by zakneutron** [comment]
Thanks for the update. I'll try this again...

**Comment by fingoldo** [comment]
> Thanks for the update. I'll try this again...

Any luck? Having similar problem now with Modin v 0.7.3 While reading a 84Gb csv file from HDD (Windows Server 2019, 128Gb RAM,16 cores), first 5 mins I am observing a ~112Mb/sec reading speed, then it drops to the 8mb/sec of regular pandas. What could be the issue?

**Comment by devin-petersohn** [comment]
@fingoldo This issue is very outdated, would you mind opening a new issue so we can help resolve your issue? It likely is not related to this issue.
